Kerpecheva/collectionview dragdrop (#882)
telerik/maui-docs: controls/collectionview/drag-and-drop/overview.md

---
title: Overview
page_title: .NET MAUI CollectionView Documentation - Drag and Drop/Reorder
description: Review the Telerik UI for .NET MAUI CollectionView Drag and Drop feature.
position: 0
slug: collectionview-draganddrop-overview
tags: reorder, collectionview, drag-drop
---

# .NET MAUI CollectionView Drag and Drop

> The .NET MAUI CollectionView provides drag and drop functionality which allows the end user to reorder its items. If the feature is enabled, when the user holds on an item, the reorder mode is triggered and the user can move and release the item at the desired position. This will perform reorder operation on the data. 

Reviewer: ```suggestion
The .NET MAUI CollectionView provides drag and drop functionality which allows the end user to reorder its items. If the feature is enabled, when the user clicks and holds an item, the reorder mode is triggered and the user can move and release the item at the desired position. This also performs a reorder operation on the data. 
```

![.NET MAUI CollectionView Reorder Items](../images/collectionview-itemsreorder.gif)

> Reorder functionality can be enabled by setting the `IsItemsReorderEnabled` property to `True`.

Reviewer: ```suggestion
To enable the reorder functionality, set the `IsItemsReorderEnabled` property to `True`.
```

> Here is a quick example of the CollectionView with reorder feature:

Reviewer: ```suggestion
Here is a quick example of a CollectionView with enabled reordering:
```
